content/en/apps/guides/messaging/gateways/rapidpro_cht_gateway.md

```diff
@@ -0,0 +1,65 @@
+#
+# RapidPro CHT APK with FCM configuration
+
+RapidPro - CHT uses your Android phone to send and receive messages on your behalf.
+
+Due to restrictions Android has on applications that send SMS messages, we cannot distribute the RapidPro - CHT application through the Play Store. You will need to download the application manually and install it using the steps below.
```

Comment:
I think it's more accurate to state that the restrictions are from Google on Android applications. I'd rephrase this a bit.
```suggestion
Due to restrictions that Google has placed on Android applications that send SMS messages, RapidPro - CHT applications can no longer be distributed through the Google Play Store. You will need to download the application file and install it using the steps below:
```